components/button/utils.tsx

```diff
@@ -5,6 +5,7 @@ import { ButtonProps } from 'components/button/button'
 import { ButtonGroupConfig } from 'components/button-group/button-group-context'
 
 export const getButtonChildrenWithIcon = (
+  loading: boolean,
```

Comment:
Is it possible to just skip this function in the button component while `loading` is `true`?

Reply:
If skip this function, in case of button with many words, loading status button width will cut down and restore after. If loading status period is short, button width flashing.